### DISCUSSION TITLE
Page Action Button Dropdown

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#2517] [Ideas]
I was thinking that it might be helpful to allow users the ability to group a bunch of page action buttons together into a dropdown of actions, like this:

<img width="343" alt="Screen Shot 2022-05-23 at 10 47 14 AM" src="https://user-images.githubusercontent.com/125735/169846098-3ec35dae-d17d-48cb-b951-b95996ed3b34.png">

My use case: I have a project where there are six different buttons and there will likely need to be more - and I'm running out of space.

Thoughts?

**Comment by danharrin** [comment]
#2272
